page.js

```diff
@@ -0,0 +1,9 @@
+function webpage(){
+	var name = document.getElementById("name");
+
+	var sentence = "Hello " + name.value;
+
+	var output = document.getElementById("line");
+	output.textContent = sentence;
```

Comment:
Lines 4, 6 and 7 might be easier to remember (and decode) if you use jQuery and ES6 format. So, instead of

``` javascript
   var sentence = "Hello " + name.value;
   var output = document.getElementById("line");
   output.textContent = sentence;
```
you would have

``` javascript
   const sentence = `Hello ${name}`;
   // By using the backticks (`) you can combine text and variable values without plus signs (+)
   // If you want to use the value of a variable, you use a dollar sign and put the variable name 
   // inside curly brackets ${}

   $("#line").html(sentence); 
   // $("#line") is getting the element by id, and .htm() is taking the place of .textContent
```